### DISCUSSION TITLE
Versión mínima de Node

### DISCUSSION THREAD

**Discussion by rodrigobustamante** [JSConfCL/tickets#12] [Q&A] [answered]
Hola! 👋
Quería consultar que versión de Node es la mínima requerida para levantar el proyecto.

Por temas de compatibilidad de las versiones de Next (14.0.3), requiere una versión mayor o igual a `18.17.0`.

Con esto ya definido, se tiene pensado usar el LTS de Node al día de hoy (`20.10.0`), o dejar como mínimo la `18.17.0`.
Mi pregunta va orientada para poder dejar un `.nvmrc` en el root del proyecto, para poder realizar el cambio de versionado más fácil y friendly.

Happy coding! 🚀

**Comment by fforres** [comment, marked as answer]
Mas que el `.nvmrc`, propongo setear el campo "engines" en el package.json @rodrigobustamante 

https://docs.npmjs.com/cli/v10/configuring-npm/package-json#engines

Es un poquito mas violento, porque tira un error si estas corriendo con una version que no soportamos.... pero al mismo tiempo más efectivo 😸 

(Por ejemplo, yo no uso NVM en el PC de desarrollo, entonces la existencia de un `.nvmrc` no cambiaría en nada mi experiencia de desarrollo.)



En mi opinion.... Usemos los 2.  `.nvmrc` mejora la experiencia de desarrollo para quien lo use, pero `engines` nos da seguridad 😄
